server/src/analyzer/expressionEvaluator.ts

```diff
@@ -849,6 +849,28 @@ export class ExpressionEvaluator {
         let validatedTypes = false;
         let returnType: Type | undefined;
 
+        if (TypeUtils.isDataClass(type)) {
+          let constructorMethodType: FunctionType = new FunctionType(FunctionTypeFlags.InstanceMethod);
+          constructorMethodType.getParameters().push({
+            category: ParameterCategory.Simple,
+            name: 'self',
+            hasDefault: false,
+            type: type
+          });
+          type.getClassFields().forEach((s, k) => {
+            if (TypeUtils.isFunctionType(s.currentType) || k[0] === '_') { return; }
```

Comment:
Minor nit: For consistency with rest of code, please move curly brackets and return statements to their own lines.